WalletWasabi.Fluent/ViewModels/Search/SearchPageViewModel.cs

```diff
@@ -45,14 +44,19 @@ public void Initialise()
 			.Select(SearchQueryFilter)
 			.DistinctUntilChanged();
 
-		_sourceObservable
-			.Filter(queryFilter)
-			.GroupWithImmutableState(x => x.Category)
-			.Transform(grouping => new SearchResult(grouping.Key, grouping.Items.OrderBy(x => x.Order).ThenBy(x => x.Title)))
-			.Sort(SortExpressionComparer<SearchResult>.Ascending(i => i.Category.Order).ThenByAscending(i => i.Category.Title))
-			.ObserveOn(RxApp.MainThreadScheduler)
-			.Bind(out _searchResults)
-			.AsObservableList();
+		if (_sourceObservable is { })
+		{
+			_sourceObservable
+				.Filter(queryFilter)
+				.GroupWithImmutableState(x => x.Category)
+						.Transform(grouping =>
+							new SearchResult(grouping.Key, grouping.Items.OrderBy(x => x.Order).ThenBy(x => x.Title)))
+						.Sort(SortExpressionComparer<SearchResult>.Ascending(i => i.Category.Order)
+							.ThenByAscending(i => i.Category.Title))
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Bind(out _searchResults)
+				.AsObservableList();
+		}
 	}
```

Comment:
Instead of wrapping it into an `if` statement you can use null propagation. `_sourceObservable?` instead of `_sourceObservable`

Reply:
done.